rdbi-recipes/src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/StateDedupedJobScheduler.java

```diff
@@ -34,10 +35,11 @@ public StateDedupedJobScheduler(RDBI rdbi, String redisPrefixKey) {
      */
     @Override
     public boolean schedule(final String tube, final String jobStr, final int runInMillis) {
-
         try (Handle handle = rdbi.open()) {
             return 1 == handle.attach(StateDedupedJobSchedulerDAO.class).scheduleJob(
                     getReadyQueue(tube),
+                    getRunningQueue(tube),
```

Comment:
i think i need to understand the logic here. usually we'd just put the job in readyQueue not directly in running queue.